zOSMF-Samples/workflow_sample_wf2wf.xml

```diff
@@ -0,0 +1,203 @@
+		and transfer its value to demoString in called workflow.
+		
+		Note: When user specify the XML definition and create the called workflow successfully, the variables that meet the mapping rules will be transferred
+		from the calling workflow to the called workflow only once, that means, no matter how many times the user perform the calling step in future, those 
+		variable values will not be transferred any more. But if user decide to change the existing called workflow, this will be treated as creating a new called
+		workflow, so the mapped variables will be transferred again in this case.
+ -->
+<step name="varMappingFromCalling2Called">
+	<title>Variable Mapping Step1</title>
+	<description>The variable String1,String2 and sampleString will be transferred from the calling workflow to the called workflow.</description>
+	<variableMapping>
+		<!-- Under the element fromCallingToCalled, all variables satisfy below conditions will be copied to the called workflow:
+				- Variable name is made up with either characters or numbers
+				- Length of the variable less than 10 characters.
+				Additionally, the value of variable "sampleString" in calling workflow will also be copied to the variable named demoString in called workflow.
+				That means a union of those variables that either satisfy the regular expression or indicated by element variableName will be copied
+				from calling workflow to called workflow.
+		-->
+		<fromCallingToCalled>
+			<regExpression>[a-zA-Z0-9]{1,10}</regExpression>
+			<variableName mapTo="demoString">sampleString</variableName>
+		</fromCallingToCalled>
+	</variableMapping>
+	<callingStepWeight>3</callingStepWeight>
+	<callingStepSkills>System Programmer</callingStepSkills>
+	<calledWorkflowDefinitionFile>/usr/lpp/zosmf/samples/workflow_sample_calledwfVarMapping1.xml</calledWorkflowDefinitionFile>
+	<calledWorkflowDescription>The workflow definition file more likely locates under directory: /usr/lpp/zosmf/samples.</calledWorkflowDescription>
+	<calledWorkflowID>workflow.sample.calledwfVarMapping1</calledWorkflowID>
+	<calledWorkflowVersion>1.0</calledWorkflowVersion>
+</step>
+ 
+<!-- This step demonstrates the use of variableMapping element for user to transfer variable values from called workflow to calling workflow.
+		The element <variableName> under the element <fromCalledToCalling indicates that when the called workflow become Complete status, 
+		the variable "userID" in the called workflow will be mapped to the variable "st_user" in the calling workflow, additionally, the variable
+		value will also be set to st_user in the calling workflow when the called workflow become to Complete status.
+		
+		Note: When the called workflow is performed and become Complete status, the variables that meet the mapping rules will be transferred
+		from the called workflow to the calling workflow, this will also happen only once no matter how the status of the called workflow change
```

Comment:
```suggestion
		from the called workflow to the calling workflow, this will also happen only once - even if the state of the called workflow changes later on.
```